frontend/rust-lib/lib-infra/src/box_any.rs

```diff
@@ -12,6 +13,13 @@ impl BoxAny {
     Self(Box::new(value))
   }
 
+  pub fn clone_content<T>(&self) -> Option<T>
```

Comment:
How about renaming it to cloned()? Lots of Rust foundation type support cloned when you want to clone the inner

Reply:
Done